src/Cardano/TextEnvelope.purs

```diff
@@ -40,13 +42,15 @@ derive instance Eq TextEnvelopeType
 instance Show TextEnvelopeType where
```

Comment:
I just realized this isn't a great `Show` instance since it doesn't evaluate to the corresponding Purescript code (that's not related to your changes though)